spec/factories/project.rb

```diff
@@ -1,7 +1,36 @@
 FactoryBot.define do
   factory :project do
+    association :author, factory: :user
     project_access_type { "Private" }
     description { Faker::Lorem.sentence }
     data { Faker::Lorem.sentence }
+    factory :project1 do
+      association :author, factory: :user
+      name {"Full Adder using basic gates"}
```

Comment:
Fill these faker, If you want this name for matching, specifiy that explicitly in test.

Like 
```ruby
@project1 = FactoryBot.create(:project, name: 'BaSic gaTe')
get search_path(query: 'basic')
expect(response.body).to include 'BaSic gaTe'
```

So that what is being tested will be obvious in the first look

Reply:
Done